packages/server/modules/core/services/branch/management.ts

```diff
@@ -95,7 +95,21 @@ export const updateBranchAndNotifyFactory =
       throw new BranchUpdateError('Please specify a property to update')
     }
 
-    const newBranch = await deps.updateBranch(input.id, updates)
+    let newBranch: BranchRecord
+    try {
+      newBranch = await deps.updateBranch(input.id, updates)
+    } catch (e) {
+      if (ensureError(e).message.includes('branches_streamid_name_unique')) {
```

Comment:
noticed that if u rename a branch to a name that already exists, the user gets an ugly SQL error